### ISSUE TITLE
Clicking 'Ghosts Ahoy' crashes plugin (NullPointerException)

### ISSUE THREAD

**Issue by NotTeeZee** [Zoinkwiz/quest-helper#66]
Reproduction:
1) Look for quest 'Ghosts Ahoy'
2) Click the arrow '>' to open quest in the Quest Helper
3) Nothing happens, unable to open any other quests from this point.

Stacktrace:

```2020-11-14 22:47:54 [Client] ERROR net.runelite.client.RuneLite - Uncaught exception:
io.reactivex.rxjava3.exceptions.UndeliverableException: The exception could not be delivered to the consumer because it has already canceled/disposed the flow or the exception has nowhere to go to begin with. Further reading: https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling | java.lang.reflect.InvocationTargetException
	at io.reactivex.rxjava3.plugins.RxJavaPlugins.onError(RxJavaPlugins.java:367)
	at io.reactivex.rxjava3.internal.observers.LambdaObserver.onError(LambdaObserver.java:83)
	at io.reactivex.rxjava3.internal.observers.LambdaObserver.onNext(LambdaObserver.java:67)
	at io.reactivex.rxjava3.internal.operators.observable.ObservableDoFinally$DoFinallyObserver.onNext(ObservableDoFinally.java:80)
	at io.reactivex.rxjava3.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:63)
	at io.reactivex.rxjava3.internal.operators.observable.ObservableFilter$FilterObserver.onNext(ObservableFilter.java:52)
	at com.jakewharton.rxrelay3.PublishRelay$PublishDisposable.onNext(PublishRelay.java:179)
	at com.jakewharton.rxrelay3.PublishRelay.accept(PublishRelay.java:144)
	at com.jakewharton.rxrelay3.SerializedRelay.accept(SerializedRelay.java:57)
	at net.runelite.client.eventbus.EventBus.post(EventBus.java:115)
	at net.runelite.client.callback.Hooks.clientMainLoop(Hooks.java:193)
	at client.draw(Client.java:1519)
	at client.ak(Client.java)
	at ba.d(GameShell.java:361)
	at ba.run(GameShell.java:315)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.reflect.InvocationTargetException: null
	at jdk.internal.reflect.GeneratedMethodAccessor362.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at net.runelite.client.plugins.Plugin.lambda$findSubscriptions$1(Plugin.java:102)
	at io.reactivex.rxjava3.internal.observers.LambdaObserver.onNext(LambdaObserver.java:63)
	... 13 common frames omitted
Caused by: java.lang.NullPointerException: null
	at com.questhelper.quests.ghostsahoy.DyeShipSteps.setupSteps(DyeShipSteps.java:260)
	at com.questhelper.steps.DetailedOwnerStep.<init>(DetailedOwnerStep.java:55)
	at com.questhelper.quests.ghostsahoy.DyeShipSteps.<init>(DyeShipSteps.java:72)
	at com.questhelper.quests.ghostsahoy.GhostsAhoy.setupSteps(GhostsAhoy.java:336)
	at com.questhelper.quests.ghostsahoy.GhostsAhoy.loadSteps(GhostsAhoy.java:96)
	at com.questhelper.questhelpers.BasicQuestHelper.startUp(BasicQuestHelper.java:47)
	at com.questhelper.QuestHelperPlugin.startUpQuest(QuestHelperPlugin.java:491)
	at com.questhelper.QuestHelperPlugin.onGameTick(QuestHelperPlugin.java:211)
	... 18 common frames omitted```

**Comment by Zoinkwiz** [comment]
Hey, I've not been able to re-produce this, would you mind providing a few more details?

1. Where were you in the quest when you tried to start up the helper and it crashed?
2. If you were mid-way through, where were you standing?
3. Did this happen only once, or was it consistently re-produced whenever you tried to start the quest?

**Comment by NotTeeZee** [comment]
I completed the quest so it doesn't show up in the Quest Helper anymore, and cannot reproduce it myself anymore either. 
I was on version 1.1.4, did not start the quest, were standing in Morytania, next to the Ectofunctus altar.
It happened every time I tried to select the quest, had to restart the client in order to get the plugin working again.
I appreciate the response, but if its an issue on my end then that's on me. Thanks for your amazing plugin!
